src/worldmap.cc

```diff
@@ -1158,6 +1161,13 @@ int wmWorldMap_load(File* stream)
     int numCities;
     if (fileReadInt32(stream, &numCities) == -1) return -1;
 
+    if (gCitiesLimitFix && numCities != wmMaxAreaNum) {
+        debugPrint("WorldMap Error: Cities limit fix is enabled, "
+                   "but the number of cities in the save file is different from "
+                   "the number of cities in the worldmap.txt file.");
+        return -1;
```

Comment:
What if the game has more cities than save file. For example, the mod was updated but it just added new cities. Is it possible to change this condition to `numCities > wmMaxAreaNum` to support this case?

Reply:
What will happen now if `numCities > wmMaxAreaNum`?

Reply:
I have not tested, but I have a gut feeling that it will be fine. `SFall` have this patch enabled by default and nothing happened.
Let's maybe keep a warning there